perfmetrics/scripts/presubmit_test/pr_perf_test/build.sh

```diff
@@ -74,11 +74,18 @@ function install_requirements() {
   # We are building fio from source because of issue: https://github.com/axboe/fio/issues/1640.
   # The fix is not currently released in a package as of 20th Oct, 2023.
   # TODO: install fio via package when release > 3.35 is available.
+  # The sed command below is to address internal bug#309563824. 
```

Comment:
Can we create a new script build_and_install_fio.sh and use that script at both the places?

Reply:
I thought of doing that too, but that made the whole thing too complex in one shot (might break things and make it difficult to debug), so I stopped. 
 I'll break this upto two PRs.

Reply:
@raj-prince addressed the merge in https://github.com/GoogleCloudPlatform/gcsfuse/pull/1506 . Please review that.